src/equations/endurance.js

```diff
@@ -1,32 +1,42 @@
-/**
-
-Arguments:
-
-@param principal CashAmount (USD or other)
-@param spendRate ($/t) CashAmount / Time
-
-@returns Time
-*/
-function endurance(principal, spendRate, totalInterest, deposits) {
-  totalInterest = 0.0;
-  deposits = 0.0;
-  return (principal+totalInterest+deposits)/spendRate;
-}
-
-/**
-
-Arguments:
-
-@param principal CashAmount (USD or other)
-@param spendRate ($/t) CashAmount / Time
-@param period Time
-
-@returns remaining CashAmount
-*/
-function remaining(principal, spendRate, period) {
-  return principal - spendRate * period;
-}
-
-module.exports = {
-  endurance, remaining
+export default class Config {
```

Comment:
Should the class be called `Endurance` as well?

Reply:
That’s a good idea. I can make that change